translate/storage/test_mozilla_lang.py

```diff
@@ -77,14 +77,14 @@ def test_multiline_comments(self):
                 "# Second comment\n"
                 "# Third comment\n"
                 ";Source\n"
-                "Target\n")
+                "Target\n\n\n")
         store = self.StoreClass.parsestring(lang)
         assert bytes(store).decode('utf-8') == lang
 
     def test_template(self):
         """A template should have source == target, though it could be blank"""
         lang = (";Source\n"
-                "Source\n")
+                "Source\n\n\n")
         store = self.StoreClass.parsestring(lang)
         unit = store.units[0]
         assert unit.source == "Source"
```

Comment:
Also source for next test doesn't include the two trailing blank lines after the string.